docs/faq/can-i-host-webiny-on-my-laptop.md

```diff
@@ -0,0 +1,13 @@
+---
+id: can-i-host-webiny-on-my-laptop
+title: Can I host Webiny on my laptop?
+sidebar_label: Can I host Webiny on my laptop?
+---
+
+No, you can't.
+
+Webiny deploys various serverless resources to AWS, including but not limited to: Lambda Functions, S3 buckets, Cognito (authentication service), API Gateways, ...
```

Comment:
there shouldn't be empty elipses (...),
it should be following an etc... .
I would change this sentence to the narrative of:
We have a various serverless resources that are deployed to AWS like x, y ,z. No need to list more than like 2 or 3. Or you can say services like X require deployment...

Reply:
fixed